Python/Product/Debugger/Debugger/DebugEngine/AD7Engine.cs

```diff
@@ -561,7 +562,7 @@ public sealed class AD7Engine : IDebugEngine2, IDebugEngineLaunch2, IDebugProgra
             _breakOnException.Clear();
             _defaultBreakOnExceptionMode = (int)enum_EXCEPTION_STATE.EXCEPTION_STOP_USER_UNCAUGHT;
 
-            SetExceptionInfo(_defaultBreakOnExceptionMode, _breakOnException);
+            TaskHelpers.RunSynchronouslyOnUIThread(ct => SetExceptionInfoAsync(_defaultBreakOnExceptionMode, _breakOnException, ct));
```

Comment:
I notice that there are a lot of these peppered over the various AD7 classes. Why do we need them? These methods are all implementations of a COM interface invoked by the VS debugger, which is supposed to always invoke them on its UI thread (well, technically, in the STA apartment). So these should be no-ops. Do we actually see cases where the calls come on the background thread?